aws-android-sdk-kinesisvideo/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
     useLibrary 'org.apache.http.legacy'
 
     defaultConfig {
-        minSdkVersion 21 // android.hardware.camera2
+        minSdkVersion 24 // android.hardware.camera2
```

Comment:
The minSdkVersion for `android.hardware.camera2` is 21, so this comment no longer makes sense.  Could you update the comment to mention what requires 24?

Reply:
Yep, makes sense.

Reply:
@richardmcclellan updated